src/main/kotlin/me/ebonjaeger/perworldinventory/conversion/ConvertService.kt

```diff
@@ -0,0 +1,90 @@
+package me.ebonjaeger.perworldinventory.conversion
+
+import com.onarandombox.multiverseinventories.MultiverseInventories
+import me.ebonjaeger.perworldinventory.GroupManager
+import me.ebonjaeger.perworldinventory.service.BukkitService
+import org.bukkit.ChatColor
+import org.bukkit.GameMode
+import org.bukkit.OfflinePlayer
+import org.bukkit.command.CommandSender
+import org.bukkit.plugin.PluginManager
+import javax.inject.Inject
+
+/**
+ * Initiates conversion tasks.
+ */
+class ConvertService @Inject constructor(private val bukkitService: BukkitService,
+                                              private val convertExecutor: ConvertExecutor,
+                                              private val groupManager: GroupManager,
+                                              private val pluginManager: PluginManager)
+{
+
+    /**
+     * Conversion status.
+     *
+     * If this is true, then a conversion is currently in progress,
+     * and another conversion cannot be started.
+     */
+    var converting = false
+
+    fun runConversion(sender: CommandSender)
+    {
+        val mvinventory = pluginManager.getPlugin("Multiverse-Inventories") as MultiverseInventories
```

Comment:
I don't know the finer mechanics of Kotlin here but this might cause a NoClassDefFoundError if Multiverse-Inventories isn't loaded on the server? Can't tell for sure but I wouldn't be surprised.

Reply:
When the command is run, it checks if its installed and enabled. But it might be safer to get the MV-I instance in the command and pass that to here, instead of grabbing it again here.